hooks/wordpress/useWordpressPosts.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2023. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { WordpressService } from "../../services/WordpressService";
+import {useCallback, useEffect, useState} from "react";
+import { LogService } from "../../../core/LogService";
+import { WordpressPostDTO } from "../../../core/wordpress/dto/WordpressPostDTO";
+import { routeValidation } from "./routeValidation";
+
+const LOG = LogService.createLogger('useWordpressPosts');
+
+export function useWordpressPostsList(url:string):
```

Comment:
We should add basic documentation about the purpose and usage to any function we make